app/testOnly/controllers/FeatureSwitchController.scala

```diff
@@ -54,7 +54,7 @@ class FeatureSwitchController @Inject()(featureSwitchView: FeatureSwitchView,
 
   def show(): Action[AnyContent] = Action.async { implicit user =>
     featureSwitchService.getAll.flatMap { featureSwitches =>
-      val fss = featureSwitches.map(x => {
+      val fss = featureSwitches.filter(_.name.name != InvalidFS.name).map(x => {
```

Comment:
should be fine to add this temp FS to staging, this is our internal tooling, we just need to drop this FS before next Show and Tell;

Reply:
It doesn't show up on the test-only feature switch page, so they don't need to ever know about it

Reply:
FeatureSwitch is out internal tool, not in prod for sure; should be fine to show it; but I guess really up to you